### PR TITLE
RELATED: RAIL-4496 improve READMEs

### PR DESCRIPTION
**Pull request by no23reason**
Make the main README more approachable and easier to scan.
Add badges to package READMEs for packages released to NPM.

---

Supported PR commands:

| Command                  | Description             |
| ------------------------ | ----------------------- |
| `ok to test`             | Re-run standard checks  |
| `extended test`          | BackstopJS tests        |
| `extended check sonar`   | SonarQube tests         |
| `extended check cypress` | Cypress E2E tests       |
| `extended check plugins` | Dashboard plugins tests |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10027/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/11991/ : FAILURE in 24m 07s

**Review by kandl — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR3065

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/2892/ : FAILURE in 1m 55s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/3197/ : SUCCESS in 21s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/2039/ : SUCCESS in 16m 23s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/11994/ : SUCCESS in 21m 52s

File: README.md [hunk 1]
```diff
@@ -1,47 +1,61 @@
...
+        <img src="https://img.shields.io/npm/dm/@gooddata/sdk-model.svg" alt="GoodData.UI on npmcharts">
+    </a>&nbsp;
+    <img src="https://img.shields.io/badge/typescript-first-blue?logo=typescript" alt="GoodData TypeScript support badge">
+    &nbsp;
+    <a href="https://www.gooddata.com/slack">
+        <img src="https://img.shields.io/badge/slack-community-green?logo=slack" alt="GoodData Community Slack">
+    </a>&nbsp;
+</p>
+
+[GoodData.UI](https://sdk.gooddata.com/gooddata-ui) is a TypeScript framework for building analytical applications on top of the [GoodData platform](https://help.gooddata.com/doc/enterprise/en), [GoodData Cloud](https://www.gooddata.com/developers/cloud-native/doc/cloud/) or [GoodData.CN](https://www.gooddata.com/developers/cloud-native/doc/cloud/).
```
Comment (andriichumak) [inline README.md:25]: @no23reason a little weird that both links are going to the same docs. Perhaps we could combine them?
```
[GoodData Cloud or GoodData.CN](https://www.gooddata.com/developers/cloud-native/doc/cloud/)
```

File: README.md [hunk 2]
```diff
@@ -1,47 +1,61 @@
...
+
+GoodData.UI consists of multiple libraries with clear-cut responsibilities ranging from low-level REST API clients up to visualization libraries that deliver React-based components to render different types of charts and tables.
+
+## Documentation and examples
+
+Learn about GoodData.UI:
+
+-   [📚 Official documentation](https://sdk.gooddata.com/gooddata-ui)
```
Comment (andriichumak) [inline README.md:33]: I would expect that this link leads to https://sdk.gooddata.com/gooddata-ui/docs/about_gooddataui.html

It's useful to show some high level concepts before diving to the docs, so I get why you did it. We are working on improvements to the landing page of the documentation, so we'll be able to update this link in the future.

**Pull request merged**